webapp/repos.py

```diff
@@ -56,6 +56,8 @@ def _filter_modified_since(self, repos_to_process, modified_since_dt):
         filtered_repos_to_process = []
         for label in repos_to_process:
             for repo_id in self.cache.repolabel2ids.get(label, []):
+                if label in filtered_repos_to_process:
```

Comment:
seems working but shouldn't this condition be rather in parent for cycle?

this way it's going to do many unnecessary loops

Reply:
Yes, done